index.html

```diff
@@ -0,0 +1,306 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Thálisson G. | Portfólio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Asap&family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="shortcut icon" href="./images/favicon-2.png" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <div id="header-block">
+
+        <!-- Foto/Avatar -->
+        <div id="avatar-block">
+            <img src="./images/meu-avatar.jpeg" alt="avatar" class="avatar">
+            <a href="#" class="avatar-block-name"><h1>Thálisson Gonçalves</h1></a>
```

Comment:
normalmente nao e usado um h1 dentro de um link, seria bom voce inverter isso, ou entao mudar de h1 para span e estilizar este span com uma classe "name". 

```suggestion
           <h1> <a href="#" class="avatar-block-name">Thálisson Gonçalves</a></h1>
```

Reply:
Feito.